### PR TITLE
Support parsing `CREATE ROLLBACK SEGMENT` in Oracle

### PR DESCRIPTION
**Pull request by chakkk309**
Ref https://github.com/apache/shardingsphere/issues/6480.

- Add `CREATE ROLLBACK SEGMENT` grammar
- Support parsing `CREATE ROLLBACK SEGMENT` in oracle
- Add test

### CODE REVIEW

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/statement/ddl/CreateRollbackSegmentStatement.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+package org.apache.shardingsphere.sql.parser.sql.common.statement.ddl;
+
+import lombok.ToString;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
+
+/**
+ * Create rollback segment statement.
+ */
+@ToString
+public class CreateRollbackSegmentStatement extends AbstractSQLStatement implements DDLStatement {
```
Comment (strongduanmu) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/statement/ddl/CreateRollbackSegmentStatement.java:27]: Are there any other dialect implementation classes for CreateRollbackSegmentStatement ? If only Oracle supports this syntax, then this abstract class is not needed.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/ddl/OracleCreateRollbackSegmentStatement.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import lombok.ToString;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.ddl.CreateRollbackSegmentStatement;
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.oracle.OracleStatement;
+
+/**
+ * Create rollback segment statement.
+ */
+@ToString
+public class OracleCreateRollbackSegmentStatement extends CreateRollbackSegmentStatement implements OracleStatement {
```
Comment (strongduanmu) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/ddl/OracleCreateRollbackSegmentStatement.java:28]: Please add final for OracleCreateRollbackSegmentStatement.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/ddl/OracleCreateRollbackSegmentStatement.java (hunk 2 above)
Comment (chakkk309) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/ddl/OracleCreateRollbackSegmentStatement.java:28]: Done.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/statement/ddl/CreateRollbackSegmentStatement.java (hunk 1 above)
Comment (chakkk309) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/statement/ddl/CreateRollbackSegmentStatement.java:27]: OK, I have deleted it.

**Pull request merged**